.travis.yml

```diff
@@ -7,7 +7,7 @@ php:
   - hhvm
 
 env:
-  - SYMFONY_VERSION="~2.3||~3.0"
+  - SYMFONY_VERSION="2.7.*"
```

Comment:
by default, it should run an unmodified Symfony constraint IMO, to ensure you get the constraints right (and you should debug why you were getting 2.6.5 for this constraint btw)